cmd/vaults/commands.go

```diff
@@ -625,7 +625,7 @@ func newRetrieveCommand() *cli.Command {
 				Usage:       "Retrieves from cache by setting this flag",
 				DefaultText: "current directory",
 				Destination: &cache,
-				Value:       true,
```

Comment:
unrelated to this PR, but the current "DefaultText" here is not too relevant here

Reply:
good catch! but looks like we'll be removing the flag and just rely on cache by default, and if not available cold store